### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by Nexusdog** [shadowfacts/ShadowMC#28]
https://pastebin.com/fUh3hmXf

**Comment by Gemta** [comment]
Can confirm this crash. Happens on launch with Minecraft 1.12.2 and ShadowMC 3.8.3.
I do however use Minecraft forge version 14.23.1.2575

Crash log:
https://pastebin.com/jMFrKfwC

**Comment by Sunconure11** [comment]
https://pastebin.com/9RQu2dVf

Can confirm it as well.

**Comment by Aphyxia** [comment]
Same error: https://pastebin.com/XPM8Gqve

**Comment by saphsara** [comment]
same issue using forge 14.23.1.2556

**Comment by dmodoomsirius** [comment]
https://gist.github.com/dmodoomsirius/51199b5a12443fcff337d2207e2a385c

**Comment by MokahTGS** [comment]
Our crash log: https://pastebin.com/2xqM77vA

Shadow MC v3.8.3
Forge 2582
Forgelin v1.6.0
Extra Rails v1.3.0

**Comment by WonderDog** [comment]
Same Crash [https://pastebin.com/uY6S98sq](url)
Forge 2568

**Comment by Sunconure11** [comment]
@shadowfacts

**Comment by shadowfacts** [comment]
I am aware. Downgrade back to 3.8.0 until this is fixed.

**Comment by Dayton4Kitsune** [comment]
Yep I get it too https://pastebin.com/Je93Vi2q (Sorry I made a new issue first, I'd delete if I knew how)
